CHANGELOG.md

```
@@ -1,10 +1,649 @@
-## Unreleased
+## 2025.4.1
+
+### General
+- Feat: アップロード可能な最大ファイルサイズをロールごとに設定可能に
+	- デフォルトで10MBになっています
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 1

(MD007, ul-indent)